### ISSUE TITLE
Excluding Air bug

### ISSUE THREAD

**Issue by Plutoren** [Efnilite/Skematic#9]
Basically I was pasting air to clear an arena but noticed it wasn't pasting the air itself since the old arenas were not clearing correctly. I used this line:
```
paste schematic "/home/snap/plugins/WorldEdit/schematics/air.schematic" at {_loc}
```
I was about to give up when I noticed that I was using the line 
```
paste schematic "/home/snap/plugins/WorldEdit/schematics/bw_military.schematic" at {_loc} excluding air
```
to paste the arena. The arena pasted in and actually included air, while the first line did not. I changed the first line to this:
```
paste schematic "/home/snap/plugins/WorldEdit/schematics/bw_air_2.schematic" at {_loc} excluding air
```
and that fixed my problem.

So perhaps excluding air is flip flopped?

Skript 2.3.5
Paper 1.12.2
FAWE 19.01.12
Skematic 1.4

I was asked to post this even though I am on an outdated version.

**Comment by Efnilite** [comment]
Note that you should update to release 2.2 and try it out there, if it's still inverted please say so, if not you can close the issue. (sorry for being unclear ^-^)

**Comment by Efnilite** [comment]
Fixed on 2.2b
